lib/util/conflicter.js

```diff
@@ -165,7 +170,41 @@ Conflicter.prototype._ask = function (file, cb) {
     }
 
     if (result.action === 'diff') {
-      this.adapter.diff(fs.readFileSync(file.path, 'utf8'), file.contents.toString());
+      var headerSize = 512;
+      var fileDescriptor = fs.openSync(file.path, 'r');
+      var existingHeader = new Buffer(headerSize);
+      try {
+        var size = fs.readSync(fileDescriptor, existingHeader, 0, existingHeader.length, 0);
+        if (size < headerSize) {
+          existingHeader = existingHeader.slice(0, size);
+        }
+      } finally {
+        fs.closeSync(fileDescriptor);
+      }
```

Comment:
Maybe you could merge and release this one then : https://github.com/sindresorhus/read-chunk/pull/2